### ISSUE TITLE
"Error: w and h must be numbers" on v4.7.8

### ISSUE THREAD

**Issue by ssugiyama** [haydenbleasel/favicons#147]
Hello! 
I use favicons via favicons-webpack-plugin.  After updating to v4.7.8,  the following error occurred.

v4.7.7 is ok. 

<pre>
./node_modules/.bin/webpack -p
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: w and h must be numbers
    at Jimp.throwError (/var/www/node_modules/jimp/index.js:82:43)
    at Jimp.contain (/var/www/node_modules/jimp/index.js:1751:27)
    at Object.resize (/var/www/node_modules/favicons/helpers-es5.js:221:27)
    at /var/www/node_modules/favicons/es5.js:60:41
    at fn (/var/www/node_modules/async/lib/async.js:746:34)
    at /var/www/node_modules/async/lib/async.js:1213:16
    at /var/www/node_modules/async/lib/async.js:166:37
    at /var/www/node_modules/async/lib/async.js:706:43
    at /var/www/node_modules/async/lib/async.js:167:37
    at Jimp.<anonymous> (/var/www/node_modules/async/lib/async.js:1209:30)
    at /var/www/node_modules/jimp/index.js:289:27
    at .<anonymous> (/var/www/node_modules/pngjs/lib/png.js:74:7)
    at g (events.js:286:16)
    at emitOne (events.js:96:13)
    at emit (events.js:188:7)
    at .<anonymous> (/var/www/node_modules/pngjs/lib/png.js:36:10)
    at emitOne (events.js:96:13)
    at emit (events.js:188:7)
    at ParserAsync._complete (/var/www/node_modules/pngjs/lib/parser-async.js:109:8)
    at emitOne (events.js:96:13)
    at emit (events.js:188:7)
    at _filter.Filter.complete (/var/www/node_modules/pngjs/lib/filter-parse-async.js:19:12)
    at Filter._reverseFilterLine (/var/www/node_modules/pngjs/lib/filter-parse.js:169:10)
    at ChunkStream._processRead (/var/www/node_modules/pngjs/lib/chunkstream.js:174:13)
    at ChunkStream._process (/var/www/node_modules/pngjs/lib/chunkstream.js:193:14)
    at ChunkStream.write (/var/www/node_modules/pngjs/lib/chunkstream.js:61:8)
    at Inflate.ondata (_stream_readable.js:556:20)
    at emitOne (events.js:96:13)
    at Inflate.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
</pre>

**Comment by dangreen** [comment]
@ssugiyama can you show your config ?

**Comment by dangreen** [comment]
@haydenbleasel you forgot compile index.js to es5.js

**Comment by dangreen** [comment]
@haydenbleasel you can add gulp execution in package.json:
```
...
  "scripts": {
    "test": "cd test && node test.js && gulp",
    "prepublish": "gulp",
    // or
    "postinstall": "gulp"
  },
...
```

**Comment by CapitalCloud** [comment]
var favicons = require('favicons/es5'); to var favicons = require('favicons');

this error was gone!

anything help?

**Comment by haydenbleasel** [comment]
Alright I'll cover this fix. One sec.

**Comment by haydenbleasel** [comment]
Alright published `favicons@4.8.1`. See if this has fixed the issue for you :)

**Comment by ssugiyama** [comment]
Works fine! Thank you so much!
